testing/xfd/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=xfd
+pkgver=1.1.2
+pkgrel=0
+pkgdesc="Xorg font display application."
+url="https://xorg.freedesktop.org/"
+arch="all"
+license="custom"
+makedepends="
+	fontconfig-dev
+	gettext-dev
+	libintl
+	libxaw-dev
+	libxft-dev
+	libxmu-dev
+	libxrender-dev
+	"
+subpackages="$pkgname-doc"
+source="
+	https://xorg.freedesktop.org/archive/individual/app/$pkgname-$pkgver.tar.gz
+	"
+#builddir="$srcdir/"
+
+build() {
+	cd "$builddir"
+	export LIBS="$LIBS -lintl"
+	./configure --prefix=/usr
+	make
+}
+
+check() {
+	cd "$builddir"
+	make check
```

Comment:
It does not appear that make check does anything usefull:

> Making check in man
make[1]: Entering directory '/home/build/aports/testing/xfd/src/xfd-1.1.2/man'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/build/aports/testing/xfd/src/xfd-1.1.2/man'
make[1]: Entering directory '/home/build/aports/testing/xfd/src/xfd-1.1.2'
make[1]: Leaving directory '/home/build/aports/testing/xfd/src/xfd-1.1.2'

If so, it's better remove it then giving the idea that this is executing a test suite.